dotnet/extensions
# [release/2.1] Add CodeQL3000 to extensions-ci-official

[release/2.1] Move to latest KoreBuild

[release/2.1] Remove unused non-Windows YAML

[release/2.1] Grab binary logs in CI builds
- add `continueOnError: true` to ensure all cleanup is done
- upload log files even in builds from forks

[release/2.1] Add CodeQL3000 to extensions-ci-official
- add a new biweekly schedule to keep database and tools up to database
- add CodeQL3000 variables
- add CodeQL3000 tasks
  - not auto-injected because that happens only in builds of the default branch
- add a tsaoptions.json file
  - cribbed values from eng/sdl-tsa-vars.config file (in other branches)
- add tag indicating CodeQL3000 tasks did useful work
  - ignore no-op task executions
